apps/docs/content/guides/realtime/concepts.mdx

```diff
@@ -1,116 +1,75 @@
 
-## Presence
+The connections are:
 
-Presence can be used to share an individual's state with others within a Channel.
+- **Migrations**: Temporarily two connections when we need to run database migrations
+- **Authorization**: Configurable connection pool to check authorization policies on join
+- **Postgres Changes**: 3 connection pools required
+  - **Subscription management**: To manage the subscribers to Postgres Changes
+  - **Subscription cleanup**: To cleanup the subscribers to Postgres Changes
+  - **WAL pull**: To pull the changes from the database
 
-```js
-import { createClient } from '@supabase/supabase-js'
-const supabase = createClient('https://<project>.supabase.co', '<sb_publishable_... or anon key>')
-const roomOne = supabase.channel('room-one') // set your topic here
+The number of connections varies based on the instance size and your configuration.
 
-// ---cut---
-const presenceTrackStatus = await roomOne.track({
-  user: 'user-1',
-  online_at: new Date().toISOString(),
-})
-```
+### Replication slots
 
-{/* supa-mdx-lint-disable-next-line Rule004ExcludeWords */}
-Each client maintains their own state, and this is then combined into a "shared state" for that Channel topic. It's commonly used for sharing statuses (e.g. "online" or "inactive"). The neat thing about Presence is that if a client is suddenly disconnected (for example, they go offline), their state is automatically removed from the shared state. If you've ever tried to build an “I'm online” feature which handles unexpected disconnects, you'll appreciate how useful this is.
-When a new client subscribes to a channel, it will immediately receive the channel's latest state in a single message because the state is held by the Realtime server.
+Realtime also uses, at maximum, 2 replication slots.
 
-## Postgres Changes
+- **Broadcast from database**: To broadcast the changes from the database to the clients
+- **Postgres Changes**: To listen to changes from the database
 
-The Postgres Changes extension listens for database changes and sends them to clients. Clients are required to subscribe with a JWT dictating which changes they are allowed to receive based on the database's [Row Level Security](/docs/guides/database/postgres/row-level-security).
+### Schema and tables
 
-```js
-import { createClient } from '@supabase/supabase-js'
-const supabase = createClient('your_project_url', 'your_supabase_api_key')
+On the `realtime` schema we will create the following tables:
 
-// ---cut---
-const allChanges = supabase
-  .channel('schema-db-changes')
-  .on(
-    'postgres_changes',
-    {
-      event: '*',
-      schema: 'public',
-    },
-    (payload) => console.log(payload)
-  )
-  .subscribe()
-```
+- `schema_migrations` - To track the migrations that have been run on the database from Realtime
+- `subscription` - Track the subscribers to Postgres Changes
+- `messages` - Partitioned table per day that's used for Authorization and Broadcast from database
```

Comment:
```suggestion
- `messages` - Partitioned table per day used for Authorization and Broadcast from database
```

I find it a little hard to understand what this means. Is the partioned table in the same database?

Is something like:

> Database table partitioned by day used by Authorization and Broadcast